src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -1440,6 +1450,42 @@ Convolution::ConvolutionExecutor::ConvolutionExecutor(const dnnl::primitive_desc
     }
 }
 
+Convolution::ConvolutionSumExecutor::ConvolutionSumExecutor(const dnnl::primitive_desc& pd,
```

Comment:
Looks like the only difference with common `ConvolutionExecutor` is additional `inputReorder` for dst memory. Anyway takes some time to get it from the code. I would ask you to add some description for the executor.

Reply:
Done